uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -1,16 +1,56 @@
     <string name="continue_button">Next</string>
-    <!-- Locale -->
-    <string name="karhoo_uisdk_locale">en_GB</string>
+    <string name="K0001">General request error</string>
+    <string name="K0002">Invalid request payload</string>
+    <string name="K0003">Could not read authorisation token</string>
+    <string name="K0004">Authentication is required for this path</string>
+    <string name="K0005">Missing required role for this request</string>
+    <string name="K0006">Rate limit exceeded</string>
+    <string name="K0007">Circuit breaker has triggered for this route</string>
+    <string name="K1001">Could not register user</string>
+    <string name="K1003">Could not register user (invalid request)</string>
+    <string name="K1004">Could not register user (invalid phone number)</string>
+    <string name="K1005">Could not get user details (invalid token)</string>
+    <string name="K1006">Could not get user details (user does not exist)</string>
+    <string name="K2001">Could not get address</string>
+    <string name="K3001">Could not get estimates</string>
+    <string name="K3003">Could not get estimates (could not find specified quote)</string>
+    <string name="K4001">Could not book trip</string>
+    <string name="K4002">Could not book trip - invalid request payload (require at least 1 set of passenger-details)</string>
+    <string name="K4003">Could not book trip (could not find specified quote)</string>
+    <string name="K4004">Could not book trip (attempt to book an expired quote)</string>
+    <string name="K4005">Could not book trip (permission denied)</string>
+    <string name="K4006">Could not book trip (payment pre-authorisation failed)</string>
+    <string name="K4007">Could not cancel trip</string>
+    <string name="K4008">Could not cancel trip (could not find specified trip)</string>
+    <string name="K4009">Could not cancel trip (permission denied)</string>
+    <string name="K4010">Could not cancel trip (already cancelled)</string>
+    <string name="K4011">Could not get trip</string>
+    <string name="K4012">Could not get trip (could not find specified trip)</string>
+    <string name="K4013">Could not get trip (permission denied)</string>
+    <string name="K4014">Could not book trip as agent</string>
+    <string name="K4018">Could not book trip as this quote is no longer available</string>
+    <string name="K4020">Could not book trip with the selected DMS</string>
+    <string name="K5001">Could not get estimates</string>
+    <string name="K5002">Could not get availability (no availability found within requested area)</string>
+    <string name="K5003">Could not get availability (no categories found within requested area)</string>
     <string name="kh_uisdk_about">About</string>
     <string name="kh_uisdk_add_card">ADD CARD</string>
     <string name="kh_uisdk_add_flight_details">Flight number (Optional)</string>
+    <string name="kh_uisdk_addCommentPlaceholder">Additional comments</string>
```

Comment:
Same issue as on iOS with key standardisation. Should be fine if just synced after fixing in Phrase.